cloudant-sync-datastore-core/src/main/java/com/cloudant/sync/replication/ChangesResultWrapper.java

```diff
@@ -50,9 +49,9 @@ public Object getLastSeq() {
     }
 
     public Multimap<String, String> openRevisions(int start, int end) {
-        Preconditions.checkArgument(start >= 0, "Start position must be greater or equal to zero.");
-        Preconditions.checkArgument(end > start, "End position must be greater than start.");
-        Preconditions.checkArgument(end <= this.size(), "End position must be smaller than changes feed size.");
+        Misc.checkArgument(start >= 0, "Start position must be greater or equal to zero.");
+        Misc.checkArgument(end > start, "End position must be greater than start.");
+        Misc.checkArgument(end <= this.size(), "End position must be smaller than changes feed size.");
```

Comment:
We need to correct this message too, `end position` can be equal to changes feed size.